### ISSUE TITLE
Capturing Hono exceptions with the Cloudflare Pages Sentry Plugin

### ISSUE THREAD

**Issue by iliketowritecode** [honojs/hono#938]
Hi there, i'd like to be able to use the Cloudflare Pages Sentry Plugin (https://developers.cloudflare.com/pages/platform/functions/plugins/sentry) to capture exceptions from my Hono API. I've got the following 2 files:

`functions/hello.ts`
```
import { Hono } from 'hono'
import { handle } from 'hono/cloudflare-pages'

const app = new Hono()

app.get('/hello', (c) => {
  throw new Error('Capture this error in Sentry!')
})

export const onRequest = handle(app, '/api')
```

`functions/_middleware.ts`
```
import sentryPlugin from "@cloudflare/pages-plugin-sentry";

export const onRequest: PagesFunction = sentryPlugin({
  dsn: "https://sentry.io/welcome/xyz",
});
```

When I trigger the error from the `hello.ts` function.. it doesn't see to be properly capturing it in the Sentry Pages Plugin. **I'd like to use the Sentry Pages Plugin so I can capture all my exceptions (not just from my Hono API's).** 

Any idea's how I could get the exception to properly trigger the Sentry Pages Plugin? Thanks!

**Comment by yusukebe** [comment]
Hi @iliketowritecode !

I did not try it with Pages, but how about rethrow the `Error` in `onError`:

```ts
app.onError((e, c) => {
  throw e
})
```

**Comment by iliketowritecode** [comment]
> Hi @iliketowritecode !
> 
> I did not try it with Pages, but how about rethrow the `Error` in `onError`:
> 
> ```ts
> app.onError((e, c) => {
>   throw e
> })
> ```

Tried this as well.. but unfortunately did not work either :( I just get a `Internal Server Error` message in the browser and no error is forward to Sentry.

**Comment by yusukebe** [comment]
@iliketowritecode 

I have found the cause of this problem. It may be fixed in the next version. Please wait for a little.

**Comment by iliketowritecode** [comment]
> @iliketowritecode
> 
> I have found the cause of this problem. It may be fixed in the next version. Please wait for a little.

Thank you so much. I appreciate your time! :) And sounds great!

**Comment by yusukebe** [comment]
Hi @iliketowritecode !

We've released v3.0.3. Install it and try the following code:

```ts
import { Hono } from 'hono'
import { handle } from 'hono/cloudflare-pages'

const app = new Hono().route('/api')

app.get('/hello', (c) => {
  throw new Error('Capture this error in Sentry!')
})

export const onRequest = handle(app)
```

This may work well:)
